src/bio/bio_internal.h

```diff
@@ -77,24 +77,40 @@ enum bio_bs_state {
  * blobstore for certain NVMe device.
  */
 struct bio_blobstore {
-	ABT_mutex		  bb_mutex;
-	ABT_cond		  bb_barrier;
-	struct spdk_blob_store	 *bb_bs;
+	ABT_mutex			 bb_mutex;
+	ABT_cond			 bb_barrier;
+	struct spdk_blob_store		*bb_bs;
 	/*
 	 * The xstream resposible for blobstore load/unload, monitor
 	 * and faulty/reint reaction.
 	 */
-	struct bio_xs_context	 *bb_owner_xs;
+	struct bio_xs_context		*bb_owner_xs;
 	/* All the xstreams using the blobstore */
-	struct bio_xs_context	**bb_xs_ctxts;
-	enum bio_bs_state	  bb_state;
+	struct bio_xs_context		**bb_xs_ctxts;
+	/* Device/blobstore health monitoring info */
+	struct bio_health_monitoring	*bb_dev_health;
```

Comment:
Put a struct here instead of pointer, doesn't need a separate memory allocation.